optimum/intel/openvino/quantization.py

```diff
@@ -1008,6 +1014,7 @@ def _weight_only_quantization(
     model: openvino.runtime.Model,
     quantization_config: Union[OVWeightQuantizationConfig, Dict],
     calibration_dataset: Optional[Union[nncf.Dataset, Iterable]] = None,
+    remove_kv_cache_precision_flag: Optional[bool] = True,
```

Comment:
Could you please explain to me why we need this `remove_kv_cache_precision_flag`? Are there any plans to remove it in the future when the accuracy issue with KV cache compression is fixed?

Reply:
I also did get why we need it. The logic around setting `KV_CACHE_PRECISION` to `fp16` is simple. We enable it only for FP16/FP32 models. For the rest of the cases, we don't specify these options and the runtime enabled `u8` cache by default. Thus, we should always remove `KV_CACHE_PRECISION` when we quantize the model.

Reply:
My motivation was not to remove this flag for mixed quantization the same way it is not removed for full quantization. 

If we agree to remove it in case of any type of compression/quantization, I should also remove it for full quantization. But I would like to check whisper performance/quality first before adding this change.

Reply:
I've done some checks for a couple of whisper architectures and do not observe any differences in generation speed or quality depending on kv cache precision. So I add removal of f16 kv cache precision flag to full_quantization method too.